### PR TITLE
Solve #229

### PR DESCRIPTION
**Pull request by dourouc05**
Before setting CPX_PARAM_SCRIND, check whether the user did not specify either CPX_PARAM_SCRIND or CPXPARAM_ScreenOutput (two different names, one parameter).

### CODE REVIEW

File: src/MOI_wrapper.jl [hunk 1]
```diff
@@ -49,11 +49,16 @@ mutable struct Optimizer <: LQOI.LinQuadOptimizer
...
-        # For consistency with MPB, output logs to stdout by default.
-        model.params = Dict{String, Any}("CPX_PARAM_SCRIND" => 1)
+        model.params = Dict{String, Any}()
         for (name, value) in kwargs
             model.params[string(name)] = value
         end
+
+        # For consistency with MPB, output logs to stdout by default.
+        if !("CPX_PARAM_SCRIND" in keys(model.params)) && !("CPXPARAM_ScreenOutput" in keys(model.params))
+            model.params["CPX_PARAM_SCRIND"] 1
```
Comment (odow) [inline src/MOI_wrapper.jl]: Missing `=`

File: src/MOI_wrapper.jl [hunk 2]
```diff
@@ -49,11 +49,16 @@ mutable struct Optimizer <: LQOI.LinQuadOptimizer
...
         model.env = env
-        # For consistency with MPB, output logs to stdout by default.
-        model.params = Dict{String, Any}("CPX_PARAM_SCRIND" => 1)
+        model.params = Dict{String, Any}()
         for (name, value) in kwargs
             model.params[string(name)] = value
         end
+
+        # For consistency with MPB, output logs to stdout by default.
+        if !("CPX_PARAM_SCRIND" in keys(model.params)) && !("CPXPARAM_ScreenOutput" in keys(model.params))
```
Comment (odow) [inline src/MOI_wrapper.jl]: `if !haskey(model.params, "CPX_PARAM_SCRIND") && !haskey(model.params, "CPXPARAM_ScreenOutput")`

**Comment by dourouc05** [comment]
A test like this, on the dictionary after calling the constructor?

**Comment by dourouc05** [comment]
I've rebased this PR onto master. Could it be merged?

**Pull request merged**